src/main/java/Scheduler.java

```diff
@@ -1,17 +1,18 @@
 import org.quartz.*;
 import org.quartz.impl.StdSchedulerFactory;
+
```

Comment:
If you don't make any changes in class, you should add/remove changed empty lines before you make a commit.